testnets/elystestnet/assetlist.json

```diff
@@ -33,6 +33,38 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/elystestnet/images/elys.png"
       },
       "coingecko_id": "elys"
+    },
+    {
+      "description": "Eden token definition",
+      "denom_units": [
+        {
+          "denom": "ueden",
+          "exponent": 0,
+          "aliases": [
+            "microeden"
+          ]
+        },
+        {
+          "denom": "meden",
+          "exponent": 3,
+          "aliases": [
+            "millieden"
+          ]
+        },
+        {
+          "denom": "eden",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "base": "ueden",
+      "name": "Eden",
+      "display": "eden",
+      "symbol": "EDEN",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/elystestnet/images/eden.png"
+      },
+      "coingecko_id": "eden"
```

Comment:
I'm fairly certain this coingecko id corresponds to an unrelated asset

Reply:
@JeremyParish69 should we leave this field blank until we register it there later?

Reply:
@JeremyParish69 I noticed that the `coingecko_id` field is actually optional, so I just removed it for the eden asset